subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,91 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more popular. Are you wondering why?
+
+- For **customers**, value lies in **convenience**:
+
+  - Subscriptions simplify the business process. Indeed, subscribers never have to remember to renew
+    their orders every month, which gives them the assurance that they will have everything they
+    need before they actually need it.
+  - Subscriptions help customers stay on budget.
+  - Subscriptions bring added value to the customer.
```

Comment:
```suggestion
  - Subscriptions simplify the business process. 
  - Subscribers do not need to remember to renew their orders every month, which assures they will have what 
     they need before they realize they need it.
  - Subscriptions help customers stay on budget.
```
- Subscriptions bring added value to the customer. <- the values are what the sentences describe?! So, I believe this sentence could be removed (: